src/main/java/Event.java

```diff
@@ -0,0 +1,13 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
+    public Event (String taskName, String from, String to) {
```

Comment:
Consider removing the empty space after "Event" and "("

eg. public Event(String taskName,...)